rusoto/core/src/region.rs

```diff
@@ -11,7 +11,7 @@ use std::fmt::{Display, Error as FmtError, Formatter};
 
 /// An AWS region.
 /// `CnNorth1` is currently untested due to Rusoto maintainers not having access to AWS China.
-#[derive(Copy, Clone, Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
Having to remove the `Copy` trait is one of the downsides of switching to `Local(String)` since `String` doesn't implement the `Copy` trait.